server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala

```diff
@@ -109,9 +116,10 @@ class AddressIndexRepository @Inject()(conf: AddressIndexConfigModule,
       case _ => Seq.empty
     }
 
+    // test with and without this - probably better not to have it was we have boost numbers
     val fromSourceQueryShould = args.fromsource match {
-      case "niboost" => Seq(termsQuery("fromSource","NI"))
-      case "ewboost" => Seq(termsQuery("fromSource","EW"))
+    //  case "niboost" => Seq(termsQuery("fromSource","NI"))
```

Comment:
Are these comments needed?

Reply:
The extra should query may or may not still be needed in addition to the boosts inside the multimatch (going without it for initial tests), so is commented rather than removed for the time being.